bin/storj.js

```diff
@@ -396,6 +396,9 @@ var ACTIONS = {
         return log('error', err.message);
       }
 
+      var keyring = storj.KeyRing(DATADIR, program.keypass);
```

Comment:
That will check if it's passed as an arg and if not it will prompt for the password.